docs/drpc-crd.md

```diff
@@ -1,3 +1,141 @@
 # DRPlacementControl(drpc) CRD
 
-## **Under construction**
+DRPlacementControl (DRPC) is a Kubernetes operator that runs in the ACM Hub cluster. Its main function is to manage the lifecycle of stateful applications. It implements the reconciliation loop pattern. The reconciliation process is triggered when a DRPC CR is created, updated, or deleted.
```

Comment:
I think we call it the `ramen-hub-operator` not DRPlacementControl operator. Given that the operator also reconciles DRPolicy CR and who knows what else in future, calling it DRPC operator may be misleading. Do you mean to say `DRPC is a Kubernetes CRD that is installed in the OCM hub cluster`?